playbooks/dependencies-fedora.yaml

```diff
@@ -39,6 +39,22 @@
       - udisks2
     use: "{{ 'dnf' if zuul.attempts > 1 else 'auto' }}"
 
+- name: Ensure subuid/subgid ranges exist for zuul-worker (workaround for Rawhide CI issue)
+  become: true
+  lineinfile:
+    path: "{{ item.file }}"
+    regexp: "^zuul-worker:"
+    line: "zuul-worker:524288:65536"
+    create: yes
```

Comment:
Nitpick: maybe `true` instead of `yes`?  That's what the [documentation](https://docs.ansible.com/ansible/latest/collections/ansible/builtin/lineinfile_module.html) uses, and you have used elsewhere.  :)